db/python/tables/family.py

```diff
@@ -217,7 +275,25 @@ async def create_family(
 RETURNING id
         """
 
-        return await self.connection.fetch_val(_query, updater)
+        new_id = await self.connection.fetch_val(_query, updater)
```

Comment:
I think this and the external id creation should happen in a transaction

Reply:
Good point